Gemfile

```diff
@@ -1,35 +1,37 @@
+# frozen_string_literal: true
+
 source 'http://rubygems.org'
 ruby '~> 2.6.4'
 
-gem 'puma'
+gem 'acts_as_api'
```

Comment:
the gemfile is order dependend (which is not great)

Reply:
> the gemfile is order dependend 

Nasty! I would suggest to put whatever need to be on top (or bottom) there and add a comment and sort the other gems alphabetically in their groups.

Reply:
@mediafinger Yea that's what I meant, like which of the change causes this a problem

Reply:
As far as I am seeing this gemfile with the current order still works properly on CI without issues

Reply:
looks like that was somehow magically resolved over time